rock.osdeps

```diff
@@ -367,8 +367,8 @@ unzip:
 
 libqglviewer:
     ubuntu:
-        '12.04,12.10,13.04': libqglviewer-qt4-dev
-        default: libqglviewer-dev
+        '12.04,12.10,13.04,13.10,14.04,14.10,15.04,15.10': libqglviewer-dev
```

Comment:
Let's try again ...

Between 12.04 and 13.04 the name was libqglviewer-qt4-dev
*Before* your change, the default was libqglviewer-dev

So unless you have special information (as e.g. that libqglviewer-dev was valid between 12.04 and 13.04), please don't change the existing mappings. Thanks.